Resources/Private/Templates/Backend/Index.html

```diff
@@ -8,6 +8,8 @@
         <link rel="stylesheet" href="{f:uri.resource(path: 'Styles/Host.css', package: compiledResourcePackage, localize: false)}">
         <link rel="stylesheet" href="{f:uri.resource(path: 'Styles/HostOnlyStyles.css', package: compiledResourcePackage, localize: false)}">
         <link rel="stylesheet" href="{f:uri.resource(path: 'Styles/ReactCrop.css', localize: false)}">
+        <link rel="stylesheet" href="{f:uri.resource(path: 'Styles/codemirror.css', localize: false)}">
```

Comment:
Still @kitsunet , why do we manually add its CSS here like this, and not via webpack's CSS loader?